### DISCUSSION TITLE
Importação parcial do banco de dados

### DISCUSSION THREAD

**Discussion by SuperPorte** [RamonSilva20/mapos#1826] [Q&A]
Saudações galera!!

Estou mudando de endereço e, por consequência disso, mudando minhas finanças.
Tem como importar parcialmente o banco de dados?
Mais especificamente não importar as finanças. Queria manter o cadastro de Clientes, OSs, Produtos e Serviços apenas.
Se sim. como faço?
Realmente não quero misturar minhas finanças antigas com as pós mudança.

Valeuuuuu!!!!

**Comment by jsfachini** [comment]
Em principio tem, você deve entrar em localhost, "caso esteja usando localmente" ir na base de dados, escolher a opção **lançamentos** e apagar o conteúdo que esta em lançamentos, seleciona de 500 em 500 e vai apagando.
lembrando de fazer um bkp de segurança antes de executar esta opção.

no phpmyadmin
![lanc](https://user-images.githubusercontent.com/72340074/170367730-62be639d-3077-4165-aeb6-a42c2a731096.png)

**Reply by SuperPorte** [reply]
Opa!!!
Mas faço isso antes de exportar ou depois de importado?

Valeuuuu!!!

**Reply by jsfachini** [reply]
@bernarcom Na real tanto faz, pode importar a base completa e depois fazer a limpeza do financeiro no banco de dados.
